wandb/sdk/wandb_run.py

```diff
@@ -840,7 +840,9 @@ def use_artifact(self, artifact_or_name, type=None, aliases=None):
             name = artifact_or_name
             if type is None:
                 raise ValueError("type required")
-            public_api = public.Api()
+            public_api = public.Api(
+                {"entity": r.entity, "project": r.project, "run": self.id,}
```

Comment:
linter doesnt like this:
If you want to use the cling formatter: `tox -e format`

Reply:
Just updated-- I think the trailing comma was the problem. Got rid of it.